src/app/page.tsx

```
@@ -5,41 +5,121 @@ import { StartView } from "./StartView";
-        <ChatView
-          messages={messages}
-          chatInput={chatInput}
-          setChatInput={setChatInput}
-          handleSendMessage={handleSendMessage}
-        />
-      )}
-    </div>
-  );
+    const [view, setView] = useState<View>(View.START);
+    const [chatPartner, setChatPartner] = useState<string>("");
+    const [messages, setMessages] = useState<any[]>([]);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)